src/v2/components/LocationGroupHeader.tsx

```
@@ -538,42 +580,21 @@ export const LocationGroupHeader = ({
-                        / +{players.length - maxVisiblePlayers} more
-                      </span>
-                    </>
+                <div className="flex items-center gap-2 w-full">
+                  {!isCopied && (
+                    <PlayerList
+                      players={players}
+                      maxVisiblePlayers={maxVisiblePlayers}
+                    />
                   )}
-                  <span className="ml-2 opacity-0 group-hover/players:opacity-100 transition-opacity">
+                  <span className="ml-2 transition-opacity">
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)